Request requesters, assignees, and verifiers

anze3db opened the pull request.

edofic committed:
Make request assignee db field nullable

edofic committed:
Make request assignee db field nullable

edofic committed:
Show people on requests

hyperNURb committed:
Merge remote-tracking branch 'upstream/develop' into feature/CORE-2424

* upstream/develop:
  Add a migration for threat actor -> threat
  Rename threat actor -> threat on the frontend
  Rename threat actor -> threat on the backend
  Rename threat actor -> threat in scss files
  Fix new_result.compute property
  Compute _backupStore before using it
  Add missing findOne/findAll attrs to dummy models
  Fix Cacheable static names test
  Replace deprecated zone() with utfcOffset()
  Fix script error on custom_definitions page
  Update new_result.compute
  Fix invalid selector
  Add missing findAll definitions
  Update frontend dependencies

hyperNURb committed:
Add can.map define plugin

hyperNURb committed:
Separate people list into groups

hyperNURb committed:
Add save persons functionality

edofic committed:
Show people on requests

hyperNURb committed:
Add and destory model

edofic committed:
Include relationships in request assignees info

hyperNURb committed:
Fix destorying relationship model

edofic committed:
Fix removal of request assignees

hyperNURb committed:
Fix toggling required fileds

hyperNURb committed:
Attach people list to the modal

hyperNURb committed:
Fix adding and removing user

hyperNURb committed:
Merge remote-tracking branch 'edofic/feature/CORE-2424-make-request-firstish-class-object' into feature/CORE-2424

* edofic/feature/CORE-2424-make-request-firstish-class-object:
  Show people on requests
  Make request assignee db field nullable
  Remove mockup code
  Update mustache lists for request info
  Add a check for parent_instance in tree view controller
  Add a helper for inserting variables into current scope
  Move info pane options out of tree view options
  Fix server side error in /search
  Add Documentable to Request
  Disable comment buttons for now
  Fix comment_list template
  Fix mapped_objects template
  Update request info.mustache
  Add new templates
  Fix request model
  Fix request mappings
  Add support for loading tree views in info controller
  Add title, description, name, and email to req subtree

hyperNURb committed:
Fix modal saving

hyperNURb committed:
Use people popover

hyperNURb committed:
Add assignees as mapping

anze3db committed:
Fix assignee mappings

hyperNURb committed:
Merge pull request #17 from Smotko/feature/fix-assignee

Fix assignee mappings

anze3db committed:
Merge remote-tracking branch 'urban/feature/CORE-2424-make-request-firstish-class-object' into HEAD

anze3db committed:
Move audit to the top of the request modal

anze3db committed:
Fix required fields for request

anze3db committed:
Add case insensitive option to if_equals helper

anze3db committed:
Add results to with_mapping helper

anze3db committed:
Run resolve_deferred_bindings if there are pending joins

anze3db committed:
Fix position of people list in modal

anze3db committed:
Enable editing assignees in request info pane

anze3db committed:
Add missing arguments to people groups

anze3db committed:
Handle deferred updates to assignees

anze3db committed:
Merge remote-tracking branch 'urban/feature/CORE-2424-make-request-firstish-class-object' into feature/request-assignees

anze3db committed:
Remove if !person

anze3db committed:
Don't toggle all assignee boxes at once

uskudnik commented on src/ggrc/assets/javascripts/components/people_list.js:
```diff
@@ -0,0 +1,193 @@
+        instance.attr("people", _.every(isAllowed));
+      },
+      inserted: function () {
+        // this.scope.instance.get_assignees().then(function (assignees) {
```
we can probably remove this all together?

anze3db committed:
Fix adding urls without a deferred

anze3db committed:
Remove dead code

anze3db committed:
Remove request before_save after_save events

These are now out of date. The after create will need to be rewritten
for if we want to add permissions for all the mapped people

anze3db committed:
Preload current user and audit.contact in new request form

anze3db committed:
Make sure _pending_joins get cleared

uskudnik commented on src/ggrc/assets/javascripts/models/audit_models.js:
```diff
@@ -437,42 +435,29 @@ can.Model.Cacheable("CMS.Models.Request", {
         .then(function(audit) {
-          that.attr('assignee', audit.contact);
-        });
+          console.log("audit contact", audit.contact)
```
Forgotten `console.log` statement

anze3db committed:
Remove console.log
